daemon/delete.go

```diff
@@ -82,7 +85,7 @@ func (daemon *Daemon) DeleteVolumes(volumeIDs map[string]struct{}) {
 }
 
 // Destroy unregisters a container from the daemon and cleanly removes its contents from the filesystem.
-func (daemon *Daemon) Rm(container *Container) error {
+func (daemon *Daemon) Rm(container *Container, forceRemove bool) (err error) {
```

Comment:
Ok, I can do that. That will reduce the size of changes as all the existing calls to Rm() will not have to be modified.